easybuild/easyconfigs/o/openstackclient/openstackclient-5.5.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,133 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/08
+easyblock = 'PythonBundle'
+
+name = 'openstackclient'
```

Comment:
Please use `OpenStackClient` for the name.

Reply:
@branfosj done